packages/backend/src/yeomanui.ts

```diff
@@ -547,8 +547,8 @@ export class YeomanUI {
       : "files";
     this.typesMap.set(genMeta.namespace, type);
     _.includes(genFilter.types, "tools-suite") && this.generatorsToIgnoreArray.push(genMeta.namespace);
-
-    if (typesHasIntersection && categoriesHasIntersection) {
+    const hidden = _.includes(genFilter.categories, "hidden");
```

Comment:
consider creating an enumerator of known categories to avoid typos in source code/tests.